package.json

```diff
@@ -25,9 +25,12 @@
     "@tanstack/react-query": "^5.28.8",
     "@tanstack/react-query-devtools": "^5.28.8",
     "@typescript-eslint/eslint-plugin": "^7.5.0",
+    "bindings-js": "file:./src/temp/stellar-xdr-web",
     "dompurify": "^3.0.11",
     "html-react-parser": "^5.1.10",
     "immer": "^10.0.4",
+    "lodash": "^4.17.21",
```

Comment:
should we use lodash for some of the helpers that we had previously created, for example: `isEmptyObject` & `isEmpty`